.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
Do we want to drop 3.5 here as well?

Reply:
Agree, didn't realize 3.5 support was dropped.